tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -183,8 +210,7 @@
               edpm_ovn_metadata_agent_metadata_agent_DEFAULT_metadata_proxy_shared_secret: 1234567842
               edpm_ovn_metadata_agent_DEFAULT_bind_host: 127.0.0.1
               edpm_chrony_ntp_servers:
-              - clock.redhat.com
-              - clock2.redhat.com
+              - pool.ntp.org
```

Comment:
we'll need to make this configurable

for downstream jobs it will indeed be clock.redhat.com

Reply:
I'll add a variable for it